modules/match.js

```diff
@@ -1,10 +1,7 @@
 import invariant from 'invariant'
-import createMemoryHistory from 'history/lib/createMemoryHistory'
-import useBasename from 'history/lib/useBasename'
+import createMemoryHistory from './createMemoryHistory'
+import createTransitionManager from './createTransitionManager'
 import { createRoutes } from './RouteUtils'
-import useRoutes from './useRoutes'
-
-const createHistory = useRoutes(useBasename(createMemoryHistory))
```

Comment:
not sure why `useBasename` is here since it is a no-op on the server when used like this.